contributing.md

```diff
@@ -395,19 +396,21 @@ Porting changes across branches:
   changes can be added in a backwards-compatible way (no API / ABI / behavior
   break in released branches).
 * Periodically, a maintainer will **forward-port** changes to newer release
-  branches all the way up to `master`.
-* The merge forward can be done with `git merge` in order to keep the commit history.
-  For example:
+  branches all the way up to `main`.
+* See [this list](https://github.com/ignitionrobotics/docs/blob/master/tools/branch_comparisons.md) to check if a branch needs porting.
```

Comment:
```suggestion
* See [this list](./tools/branch_comparisons.md) to check if a branch needs porting.
```

Personally, I prefer relative paths instead of GitHub links in case things like the branch name ever changes (for example, this is targeting `master` right now, but I am assuming that this will be switched to `main` later).

Reply:
So, the problem here is that when this is hosted on https://ignitionrobotics.org/docs/all/contributing, the comparison document is not served by that server. This is convenient because we can update the table on GitHub and don't need to redeploy the web server.

Reply:
I didn't realize that this was used to host the contributing guide on the ignitionrobotics website - using the full link makes sense then. I'll mark this as resolved.